creator-node/src/fileManager.js

```diff
@@ -323,7 +320,7 @@ const trackDiskStorage = multer.diskStorage({
   destination: function (req, file, cb) {
     // save file under randomly named folders to avoid collisions
     const randomFileName = getUuid()
-    const fileDir = path.join(req.app.get('storagePath'), randomFileName)
+    const fileDir = path.join(DiskManager.getTmpTrackUploadArtifactsPath(), randomFileName)
```

Comment:
so this callback is what clears the fileDir?

Reply:
@hareeshnagaraj  no this just generates the folder path and sets it in req.fileDir. the actual track_content route clears the old artifacts